code/lib/ui/src/index.tsx

```diff
@@ -1,5 +1,7 @@
 // eslint-disable-next-line @typescript-eslint/triple-slash-reference
 /// <reference path="./typings.d.ts" />
+/// <reference types="node" />
+/// <reference types="webpack-env" />
```

Comment:
Why do we need this?

Reply:
See: https://github.com/storybookjs/storybook/pull/19269#discussion_r981415077